app/views/markets/index.html.erb

```diff
@@ -0,0 +1,67 @@
+<div class='nav'>
+  <%= link_to 'home', '/', class: 'top-nav-button' %>
+</div>
+
+<hr class="nav-hr" />
+
+<div class="pg-header-container">
+  <h1 class="pg-header">farmer's markets</h1>
+</div>
+
+
+
+  <% if @buttons %>
+  <div class='three_button_nav'>
+    <%= link_to(markets_path(order: 'az'), :class => "nav_button") do %>
+      <hr />
+      a - z
+      <hr />
+    <% end %>
+    <%= link_to(new_market_path, :class => "nav_button") do %>
+      <hr />
+      add
+      <hr />
+    <% end %>
+    <%= link_to(markets_path(order: 'za'), :class => "nav_button") do %>
+      <hr />
+      z - a
+      <hr />
+    <% end %>
+    </div>
+
+  <div class='table_list'>
+    <table>
+      <% @markets.each do |market| %>
+      <tr>
+        <td><%= link_to market.name.titleize, market, class: "market-link"%></td>
+        <td><%= button_to '', edit_market_path(id: market.id), method: :get, class: 'edit-button' %></td>
```

Comment:
Do you need the `id: market.id` part within the path or would it work if just `market.id` was specified?